src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -14,6 +14,9 @@ internal static partial class GlobalizationMode
         private static bool TryGetAppLocalIcuSwitchValue([NotNullWhen(true)] out string? value) =>
             TryGetStringValue("System.Globalization.AppLocalIcu", "DOTNET_SYSTEM_GLOBALIZATION_APPLOCALICU", out value);
 
+        internal static bool GetPredefinedCulturesOnly() =>
```

Comment:
>         [](start = 0, length = 8)

please add a property here 

internal static bool PredefinedCulturesOnly { get; } = GetPredefinedCulturesOnly();

and make GetPredefinedCulturesOnly private then use PredefinedCulturesOnly in the calling sites.

or you can just define it as 

internal static bool PredefinedCulturesOnly { get; } = GetSwitchValue("System.Globalization.PredefinedCulturesOnly", "DOTNET_SYSTEM_GLOBALIZATION_PREDEFINED_CULTURES_ONLY");

the reason for not using GetPredefinedCulturesOnly directly is we need to cache the value and not evaluate it every time we call the method GetPredefinedCulturesOnly.